vms/evm/customlogs/log_ext.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package customlogs
+
+import ethtypes "github.com/ava-labs/libevm/core/types"
+
+// FlattenLogs converts a nested array of logs to a single array of logs.
+func FlattenLogs(list [][]*ethtypes.Log) []*ethtypes.Log {
+	totalLen := 0
```

Comment:
```suggestion
	numLogs := 0
```
nit/feel-free-to-ignore